libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/service/EvcsService.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.di.ipv.core.library.service;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
+import software.amazon.lambda.powertools.tracing.Tracing;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.client.EvcsClient;
+import uk.gov.di.ipv.core.library.domain.VerifiableCredential;
+import uk.gov.di.ipv.core.library.dto.EvcsCreateUserVCsDto;
+import uk.gov.di.ipv.core.library.dto.EvcsGetUserVCDto;
+import uk.gov.di.ipv.core.library.dto.EvcsUpdateUserVCsDto;
+import uk.gov.di.ipv.core.library.enums.EvcsVCState;
+import uk.gov.di.ipv.core.library.exception.EvcsServiceException;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static uk.gov.di.ipv.core.library.enums.EvcsVCState.CURRENT;
+import static uk.gov.di.ipv.core.library.enums.EvcsVCState.PENDING_RETURN;
+
+public class EvcsService {
+    private final EvcsClient evcsClient;
+
+    @ExcludeFromGeneratedCoverageReport
+    public EvcsService(ConfigService configService) {
+        this.evcsClient = new EvcsClient(configService);
+    }
+
+    public EvcsService(EvcsClient evcsClient) {
+        this.evcsClient = evcsClient;
+    }
+
+    @Tracing
+    public void storeIdentity(
+            String userId,
+            List<VerifiableCredential> credentials,
+            String evcsAccessToken,
+            boolean isF2FIncomplete)
+            throws EvcsServiceException {
+        List<EvcsVCState> vcStatesToQueryFor = List.of(CURRENT, PENDING_RETURN);
```

Comment:
This could probably be made a static constant, and then just reused for every call to this method.

Reply:
just used in this method....done